keyboards/planck/keymaps/peterfalken/keymap.c

```diff
@@ -0,0 +1,117 @@
+    _______, _______, _______, _______, _______, _______, _______, _______,    KC_MNXT,    KC_VOLD, KC_VOLU, KC_MPLY
+),
+
+/* Raise
+ * ,-----------------------------------------------------------------------------------.
+ * |   `  |   1  |   2  |   3  |   4  |   5  |   6  |   7  |   8  |   9  |   0  | Bksp |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * | Del  |  F1  |  F2  |  F3  |  F4  |  F5  |  F6  |   -  |   =  |   [  |   ]  |  \   |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |   ▽  |  F7  |  F8  |  F9  |  F10 |  F11 |  F12 |ISO # |ISO / |Pg Up |Pg Dn |   ▽  |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |   ▽  |   ▽  |   ▽  |   ▽  |   ▽  |   ▽      ▽  |   ▽  | Next | Vol- | Vol+ | Play |
+ * `-----------------------------------------------------------------------------------'
+ */
+[_RAISE] = LAYOUT_planck_grid(
+    KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_BSPC,
+    KC_DEL,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_MINS, KC_EQL,  KC_LBRC, KC_RBRC, KC_BSLS,
+    _______, KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11,  KC_F12,  KC_NUHS, KC_NUBS, KC_PGUP, KC_PGDN, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, KC_MNXT, KC_VOLD, KC_VOLU, KC_MPLY
+),
+
+/* Adjust (Lower + Raise)
+ * ,-----------------------------------------------------------------------------------.
+ * | RSET |   ▽  | M.AP | M.AL | LPAD |   ▽  |   ▽  |   ▽  |   ▽  |   ▽  |   ▽  |  Del |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |   ▽  |   ▽  |   ▽  |   ▽  |   ▽  |   ▽  |   ▽  |   ▽  |   ▽  |   ▽  |   ▽  | SLEP |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * | CPWD |  UN  |  CU  |  CO  |  PA  |  MV  |   ▽  |   ▽  |   ▽  |   ▽  |   ▽  |   ▽  |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |   ▽  |   ▽  |   ▽  |   ▽  |   ▽  |   ▽      ▽  |   ▽  |   ▽  |   ▽  |   ▽  |   ▽  |
+ * `-----------------------------------------------------------------------------------'
+ */
+[_ADJUST] = LAYOUT_planck_grid(
+    QK_BOOT, _______,   CK_M1, KC_MCTL, KC_LPAD, _______, _______, _______, _______, _______, _______,  KC_DEL,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,  CK_SLP,
+      CK_CW, _______,   CK_CU,   CK_CO,   CK_PA,   CK_MV, _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
+)
+
+};
+
+bool muse_mode = false;
+uint8_t last_muse_note = 0;
+uint16_t muse_counter = 0;
+uint8_t muse_offset = 70;
+uint16_t muse_tempo = 50;
+
+void matrix_scan_user(void) {
```

Comment:
Is the muse stuff even being used, since the encoder code isn't present?

Reply:
You're right @drashna, I removed the muse/midi processing and variables.
Thank you for the feedback and reviews.